wger/settings.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from wger.settings_global import *
+import os
+
+# Use 'DEBUG = True' to get more details for server errors
+DEBUG = True
+TEMPLATES[0]['OPTIONS']['debug'] = True
+
+ADMINS = (
+    ('Your name', 'your_email@example.com'),
+)
+MANAGERS = ADMINS
+
+
+DATABASES = {
+    'default': {
+        'ENGINE': os.environ.get('DB_ENGINE', 'django.db.backends.sqlite3'),
+        'NAME': os.environ.get('DB_NAME', '/home/paul/Workspace/wg-kalii/wger/database.sqlite'),
+        'USER': os.environ.get('DB_USER', ''),
+        'PASSWORD': os.environ.get('DB_PASSWORD', ''),
+        'HOST': os.environ.get('DB_HOST', ''),
+        'PORT': os.environ.get('DB_PORT', ''),
+    }
+}
+
+# Heroku database URL
+if 'DATABASE_URL' in os.environ:
+    import dj_database_url
+DATABASES = {'default': dj_database_url.config()}
+
+# Make this unique, and don't share it with anybody.
```

Comment:
I don't think we should leave the `SECRET_KEY` exposed like this. We can get this as an environment variable.  Check out this doc on how to use Env variables in django https://godjango.com/blog/working-with-environment-variables-in-python/.

Reply:
Hi @collin5 , the SECRET_KEY has now been hidden. It can now only be set by the person working on the project.